apps/frontend/src/components/global/TopbarDropdown.vue

```diff
@@ -118,6 +118,8 @@ export default class TopbarDropdown extends mixins(ServerMixin) {
         this.userInfo.firstName.charAt(0) +
         this.userInfo.lastName.charAt(0)
       );
+    } else if (this.userInfo.firstName){
+      return this.userInfo.firstName.charAt(0);
```

Comment:
I'd be fine with just getting the first 2 characters if we know we only have a firstname.